plugin/src/App/StatusPages/Connected.lua

```diff
@@ -83,10 +83,7 @@ function ChangesDrawer:render()
 				transparency = self.props.transparency,
 				layoutOrder = 3,
 
-				columnVisibility = { true, false, true },
-				patch = self.props.patch,
-				unappliedPatch = self.props.unappliedPatch,
-				instanceMap = self.serveSession.__instanceMap,
+				tree = self.props.patchTree,
```

Comment:
It would be ideal if the same prop passed to multiple components have consistent naming.